lib/ansible/modules/network/f5/bigip_pool_member.py

```diff
@@ -384,7 +402,8 @@ def main():
         description=dict(type='str'),
         rate_limit=dict(type='int'),
         ratio=dict(type='int'),
-        preserve_node=dict(type='bool', default=False)
+        preserve_node=dict(type='bool', default=False),
+        priority_group=dict(type='int')
```

Comment:
Thus needs adding to the DOCUMENTION block code a the top of the file, just below where preserves node is documented. 

 This PR will be released in Ansible 2.4, so please add `version_added:"2.4"`